bridge/bridgeapp/views.py

```diff
@@ -27,9 +28,13 @@ def get(self, request):
 
 class BridgeCategoryView(View):
     """Get Thread table from database and display threads associated with Category selected on Home Page"""
-    def get(self, request, category_id, category_slug):  #category_slug needed for UI
+
+    def get(self, request, category_id, category_slug):  # category_slug needed for UI
+        # assigns list of category object id's to category variable
         category = Category.objects.get(id=category_id)
+        # assigns threads filtered by category and ordered by date to threads variable
```

Comment:
Here the purpose which should be identified first is to **get all the questions/threads that pertain to the category in line 34**.

The `filter()` and `sort_by()` methods read what they do already.

You could if the comment is within a 79-word line add "in desc order" to explain the '-' sign.